spanner/client.go

```diff
@@ -867,13 +867,20 @@ func (c *Client) BatchReadOnlyTransaction(ctx context.Context, tb TimestampBound
 		err error
 	)
 
-	// Create session.
-	s, err = c.sc.createSession(ctx)
-	if err != nil {
-		return nil, err
+	if c.idleSessions.isMultiplexedSessionForPartitionedOpsEnabled() {
+		sh, err = c.idleSessions.takeMultiplexed(ctx)
```

Comment:
In case of partitioned read/query methods how is it getting multiplexed sessions? I don't see any changes in this PR for them

Reply:
For partition read/query we are using BatchReadOnlyTransaction object which already has transaction created using multiplex session so they are just reusing it.